packages/codemods/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@suspensive/codemods",
```

Comment:
I put a lot of thought into the name.

The package I’ve implemented (codemods) was intended to manage multiple packages (such as suspensive/react, react-query, jotai, and more...) in a unified way. To create a codemod that goes beyond just supporting individual packages within suspensive, I chose the name "codemods" to reflect this integrative approach.

Additionally, I thought it would be best to maintain consistency with naming conventions seen in Next.js and React documentation, where the term "Codemods" is used in descriptions and the command itself is "Codemod". This would be similar to naming conventions like "jest-codemods"

Reply:
Okay! Thanks for the explanation